e2e/e2e_suite_test.go

```diff
@@ -27,15 +27,21 @@ import (
 	"github.com/shurcooL/githubv4"
 	"go.uber.org/zap"
 
+	"github.com/ossf/scorecard/clients"
+	"github.com/ossf/scorecard/clients/githubrepo"
 	"github.com/ossf/scorecard/roundtripper"
 )
 
 var (
 	ghClient    *github.Client
 	graphClient *githubv4.Client
-	client      *http.Client
+	httpClient  *http.Client
+	repoClient  clients.RepoClient
 )
 
+// List of repos to fetch source code of.
+var repoList = []string{"ossf/scorecard", "netlify/netlify-cms", "tensorflow/tensorflow"}
```

Comment:
After some thought, this approach feels hacky.  The present implementation expects to run the scorecard binary for things to work. No other library (for example another go project) cannot refer to scorecard as a dependency and use it because the file downloading is within the main package. 

I mean checks package cannot be used independently.

Reply:
this has been removed. I've moved everything under the tests themselves to avoid sharing the tarball and concurrency issues.